jenkins-scripts/dsl/_configs_/Globals.groovy

```diff
@@ -72,7 +72,7 @@ class Globals
 
    static ArrayList get_other_supported_distros()
    {
-     return [ 'bionic' ]
+     return [ 'focal' ]
```

Comment:
Seems weird to have 'focal' in other supported distros together with ci_distro. Looking at the consumers:
 * ignition.dsl is adding both to all_supported_distros (probably duplicating the job generation)
 * gazebo.dsl add a daily SCM to all_supported_distros (unnecessary)
 * sdformat.dsl idem with gazebo.dsl

```suggestion
     return [  ]
```